templates/index.html

```diff
@@ -377,8 +377,7 @@
           <br>
           <div class="form-group">
             <label class="heading" for="TRAVIS_TAG">Name of your Distribution</label>
-            <input pattern="[A-Za-z0-9\s]{1,20}" class="form-control" placeholder="Enter name" title="Event name to trigger build"
-              name="TRAVIS_TAG" required="" type="text" data-toggle="tooltip" title="Only Aplhanumeric Characters allowed">
+            <input pattern="[A-Za-z0-9\s]{1,20}" class="form-control" placeholder="Enter name" name="TRAVIS_TAG" required="" type="text" data-toggle="tooltip" title="Only alphanumeric characters allowed">
```

Comment:
it will be confusing now cause user will not know what this field in the form is for and required for 
you can use tooltips to add info like only alphanumeric char allowed

Reply:
There are two title before this, so only the 1st one gets executed but we need to show the user the only possible input in that field.

Reply:
think like the user I don't know what to input then what is the use of what I can enter

Reply:
For that instance we are placing a placeholder which user has to enter a name for his distro

Reply:
@meets2tarun @xeon-zolt @niccokunzmann I have a suggestion. We can put an * after " Name of your Distribution" and when anyone clicks on it a message will show "Only alphanumeric characters allowed"

Reply:
Cool @rpotter12 Go ahead

Reply:
![img_500907](https://user-images.githubusercontent.com/15213246/42106002-102b7760-7bf0-11e8-914a-23a1e42da8d7.png)